plugin/www/AppboyPlugin.js

```diff
@@ -40,6 +40,18 @@ AppboyPlugin.prototype.changeUser = function (userId) {
 }
 
 /**
+* Registers the device as eligible to receive push notifications from Appboy.
+* Appboy will use the provided For GCM/ADM applications, this takes the GCM/ADM registration ID to send the device GCM/ADM messages.
+* For apps integrating Baidu Cloud Push, this method is used to register the Baidu user with Appboy.
+* This should only be used if you already use GCM/ADM messaging in your app from another provider or are integrating Baidu Cloud Push.
+*
+* @param {string} registrationId - The registration ID, or for apps integrating Baidu Cloud Push, the Baidu user id.
+*/
+AppboyPlugin.prototype.registerAppboyPushMessages = function (gcmRegistrationID) {
```

Comment:
FYI going to leave this on branch for now, but it might get modified in a nonbreaking way (changing parameter/method to be generic across platforms) or in a breaking way (changing the method to be like registerAppboyPushMessagesAndroid) to make it more clear that it's a NOOP on iOS.  I'll be sure to give notice when we make a decision/change there.